recipes-devtools/python/python3-powerrelay/labgrid-powerrelay.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Powerrelay init
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Type=simple
+Environment="LC_ALL=en_US.UTF-8"
+Environment="LANG=en_US.UTF-8"
+
+ExecStart=/bin/bash -c '/usr/bin/powerrelay run /etc/config.yaml'
+
```

Comment:
Is it strictly necessary to wrap the powerrelay bin in "/bin/bash -c" ?

Reply:
I agree with @krevsbech, the `bash -c` shouldn't be here.